index.html

```diff
@@ -28,8 +28,9 @@
       <header class="text-center">
         <h1>Welcome to Public Lab!</h1>
         <p>
-          Powered by Commmunity Toolbox
-          <a href="https://github.com/jywarren/community-toolbox"><i class="fa fa-github"></i></a> Open Source
+          Powered by <a href="https://github.com/jywarren/community-toolbox"><i class="fa fa-github"></i></a>
+          <a href="https://github.com/jywarren/community-toolbox">Commmunity Toolbox</a> 
+          | Open Source
```

Comment:
@jywarren do we need these changes?

Reply:
This part is not present in main branch. Do we need this "powered by" part? @jywarren